doc/examples/transform/plot_tps_deformation.py

```diff
@@ -0,0 +1,145 @@
+r"""
+==========================================
+Interpolate images with thin-plate splines
+==========================================
+
+Thin-plate splines (TPS) refer to a method for interpolating data [1]_.
+In an image context, we use a 2D generalization of TPS, i.e., a two-dimensional
+simulation of a 1D cubic spline, which is the solution of the biharmonic equation [2]_.
+According to [3]_, the basic solution of the biharmonic function was expanded.
+Given pairs of source and target control points, TPS is used to transform a space,
+which, in our case, is a 2D image.
+
+
+.. [1] Wikipedia, Thin plate spline
+       https://en.wikipedia.org/wiki/Thin_plate_spline
+
+.. [2] Weisstein, Eric W. "Biharmonic Equation." From MathWorld--A Wolfram Web Resource
+       https://mathworld.wolfram.com/BiharmonicEquation.html
+
+.. [3] Bookstein, Fred L. "Principal warps: Thin-plate splines and the
+       decomposition of deformations." IEEE Transactions on pattern analysis and
+       machine intelligence 11.6 (1989): 567–585.
+
+
+Image Deformation
+=================
+
+In this example, we use the TPS interpolation to deform an image.
```

Comment:
It would be nicer to *tell a story*, e.g.:

"Image deformation implies displacing the pixels of an image relatively to one another. In this example, we deform the (2D) image of an astronaut by using thin-plate splines."

The notion of interpolation/interpolator needs to be introduced earlier or separately, in any case properly. Remember that each gallery example should be self-contained.

Reply:
I agree it would indeed be nicer to tell a story. I will adopt your example.